src/test/java/de/qabel/desktop/ui/LayoutControllerTest.java

```diff
@@ -28,17 +33,30 @@ public void getQuotaStateFailuresThenHidesQuotaBars() {
     @Test
     public void testFillQuotaInformation() {
         Locale.setDefault(new Locale("te", "ST"));
+        ((BoxClientStub) boxClient).quotaState = setQuota();
         LayoutController controller = createController();
+
         assertEquals("30%", controller.quota.getText());
         assertEquals(30, (int) controller.quotaBar.getMinWidth());
-        assertThat(controller.quotaDescription.getText(), containsString("286.1 MB"));
+        assertThat(controller.quotaDescription.getText(), containsString("300 MB"));
+    }
+
+    private QuotaState setQuota() {
```

Comment:
it doesn't set the quota. you can inline this and move the code to a `setUp` method